playbooks/prep-node-disks.yml

```diff
@@ -0,0 +1,25 @@
+---
+- hosts: all
+
+  tasks:
+  - name: install dependencies
+    apt: pkg="gdisk"
+  - name: run scan_disks
+    scan_disks:
+
+  - name: check to see what drives are mounted
+    shell: mount | grep {{ item }} | awk '{print $3}'
+    with_items: ssd_devices | union(hdd_devices)
+    ignore_errors: true
+    register: mount_point
+
+  - name: unmount all mounted devices
+    shell: umount {{ item.stdout }}; sed -i '\#{{ item.stdout }}#d' /etc/fstab
+    ignore_errors: true
+    with_items: mount_point.results
+    when: item.stdout != ''
+
+  - name: remove all the partitions off all the disks
+    shell: sgdisk --zap {{ item }}
+    ignore_errors: true
+    with_items: ssd_devices | union(hdd_devices)
```

Comment:
This also seems somewhat dangerous to zap all the disks you find. Any way to narrow the target?